src/net/toxme.cpp

```diff
@@ -78,14 +134,20 @@ QByteArray Toxme::prepareEncryptedJson(int action, QString payload)
 ToxId Toxme::lookup(QString address)
 {
     // JSON injection ?
+    address = address.trimmed();
     address.replace('\\',"\\\\");
     address.replace('"',"\"");
 
     ToxId id;
     const QString json{"{\"action\":3,\"name\":\""+address+"\"}"};
     static const QByteArray pattern{"public_key\""};
```

Comment:
Hmm.. it is strange, when i test it yesterday, it did not work with "tox_id" (server return string with "public_key"